### DISCUSSION TITLE
what is the best pratice to write inline code block?

### DISCUSSION THREAD

**Discussion by SparkElf** [remarkjs/remark#988] [General] [answered]
![image](https://user-images.githubusercontent.com/58362707/166141670-3d6eeff3-5fcb-4852-b594-3cdb79f860d3.png)
the out put seems not as expected
![image](https://user-images.githubusercontent.com/58362707/166141693-8117351d-9df6-44db-9d88-095ffed2426c.png)

**Comment by Murderlon** [comment, marked as answer]
There is not enough information here to help. Are you using `react-markdown`? What would you like to happen? A reproducible example would help (see [asking quality questions](https://github.com/unifiedjs/.github/blob/main/support.md#asking-quality-questions)).
